themes/aframe/source/css/secondary.styl

```diff
@@ -172,6 +172,49 @@ a
 .section-subnav
     margin-bottom 10px
 
+.copy .post__heading
+    margin-bottom 0
+
+.post__projects
+    display flex
+    flex-wrap wrap
+    margin-bottom 15px
+    margin-left: -1px
+    a
+        background-position center
+        background-repeat no-repeat
+        background-size cover
+        border 1px solid transparent
+        height 160px
+        width 50%
+        &:hover
+            box-shadow 0 0 0 1px rgba($copy-link-hover, 0.75)
+
+.post-listing .post__projects a
+    display none
+    &:nth-child(1),
+    &:nth-child(2)
+        display block
+
+.post__projects .blog-attribution
+    border none
+    color $medium
+
+p.post__timestamp
+    font-weight 500
+    margin-right 3px
+
+.post__metadata
+    align-items center
+    display flex
+    p
+      margin-bottom 0
```

Comment:
this removes the margin on the individual post pages because there's no `<p>What's up with A-Frame for the week of Feb 19, 2016 to Feb 26, 2016.</p>`:

![image](https://cloud.githubusercontent.com/assets/203725/14481585/40aa0d72-00e9-11e6-8a07-74787d58f200.png)

to fix, add this style:

```styl
.post__projects
    margin-top 15px
```